src/main/scala/org/virtuslab/internal/load/reader/ParseCtx.scala

```diff
@@ -0,0 +1,63 @@
+package org.virtuslab.internal.load.reader
+
+import org.virtuslab.internal.load.reader
+import org.virtuslab.internal.load.reader.StateStack
+import token.Token
```

Comment:
```suggestion
import org.virtuslab.internal.load.reader.token.Token
```
IMO full path is more clear